Support for AGP 8.0.0 and above

I recently encountered a problem where if I use any Gradle Distribution above 8.0.0, the build continuously fails with the message: 

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':bdk_flutter'.
 Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
    Namespace not specified. Specify a namespace in the module's build file. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
 Run with --stacktrace option to get the stack trace.
 Run with --info or --debug option to get more log output.
 Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 23s
Error: Gradle task assembleDebug failed with exit code 1

Exited (1).


I believe the fix itself is simple enough, and a namespace needs to be added to the build.gradle file

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -26,6 +26,7 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 31
+    namespace "com.bdk.bdk_flutter"
 
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 31
-    namespace "com.bdk.bdk_flutter"
+    namespace "io.bdk.f.bdk_flutter"
 
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8

--- a/android/src/main/AndroidManifest.xml
+++ b/android/src/main/AndroidManifest.xml
@@ -1,1 +1,1 @@
-<manifest package="io.bdk.f.bdk_flutter"></manifest>
+<manifest></manifest>

